packages/cosmic-swingset/test/test-make.js

```diff
@@ -43,3 +68,92 @@ test('make and exec', async t => {
     }),
   );
 });
+
+const query = async args => {
+  const parts = [];
+  const run = new Promise((resolve, reject) => {
+    const cmd = spawn(
+      'bin/ag-chain-cosmos',
+      ['--chain-id=agoric', 'query', ...args],
+      { cwd: `${dirname}/..` },
+    );
+    cmd.stdout.on('data', chunk => parts.push(chunk));
+    cmd.on('exit', code => {
+      if (code !== 0) {
+        reject(Error(`exit ${code} from: query ${args}`));
+        return;
+      }
+      const txt = parts.join('');
+      if (txt.trim() === '') {
```

Comment:
might as well pass the trimmed version to JSON.parse()
```suggestion
      const txt = parts.join('').trim();
      if (txt === '') {
```